addons/trenches/functions/fnc_getSurfaceTexturePath.sqf

```diff
@@ -50,15 +50,15 @@ private _result = [];
 if !(isNil QGVAR(automaticFileSetup)) then {
    GVAR(automaticFileSetup) params ["_basePath", "_filePrefix"];
    _result = [_surfaceType, _basePath, _filePrefix] call _getTexturePath;
-}else{
+} else {
    private _basePath = getText (configFile >> "CfgWorlds" >> "Altis" >> "surfaceTextureBasePath");
    if ((_surfaceType find "#Gdt" == -1) || {worldName == "Tanoa"}) then {
        _basePath = getText (configFile >> "CfgWorlds" >> worldName >> "surfaceTextureBasePath")
    };
     _result = [_surfaceType, _basePath, getText(configFile >> "CfgWorlds" >> worldName >> "filePrefix")] call _getTexturePath;
 };
 
-if (isNil {_result} || _result isEqualTo []) then {
+if (isNil _result || _result isEqualTo []) then {
```

Comment:
isNil on a path? what?
Also I don't see how result can become nil here. Unless _getTexturePath can return nil. In that case, why not just make _getTexturePath return `[]`
?


Actually just checked _getTexturePath cannot return empty array. So _result will never be equal to empty array? Don't know what you are trying to check here. Doesn't really make sense.

Reply:
Well it may make no sense but it works on surfaces that create problems!